typhon/func.py

```diff
@@ -365,6 +442,10 @@ def add_parameter(self, name, _type, default=inspect._empty):
             # Add the control widget to our contents
             self.optional.contents.layout().addWidget(cntrl)
 
+        if tooltip is not None:
+            cntrl.setToolTip(tooltip)
```

Comment:
Also this sets the tooltip for the whole widget. Do we want to go with our `SignalPanel` option and put it on the label. In this case it would be `cntrl.param_label`

Reply:
👍